src/test/java/ch/uzh/ifi/hase/soprafs24/controller/CommentControllerTest.java

```diff
@@ -182,6 +194,9 @@ public void getComments_validInput_returnComments() throws Exception {
         .thenReturn(testUser);
     given(commentService.getCommentsByTaskId(Mockito.anyLong())).willReturn(List.of(comment));
 
+    // mock pusher service
+    Mockito.doNothing().when(pusherService).updateComments(Mockito.anyString());
+
```

Comment:
why? we are here in a `getComments` request, which does not use pusher. would not need to mock this (is never called anyways)

=> but again, this mock would make more sense in the `@BeforeEeach` so you never have to worry again to mock this in this test class.